jgt_tools/check_version.py

```diff
@@ -47,11 +47,12 @@ def check_default_branch(default_branch):
 def check_version():
     """Verify the version is changed if any code files are changed."""
     default_branch = find_default_branch()
-    if os.getenv("GITHUB_ACTIONS") == "true":
-        changed_files, pyproject_diff = check_default_branch(f"origin/{default_branch}")
-    else:
-        changed_files, pyproject_diff = check_default_branch(default_branch)
-
+    default_branch = (
+        f"origin/{default_branch}"
+        if os.getenv("GITHUB_ACTIONS") == "true"
+        else default_branch
+    )
+    changed_files, pyproject_diff = check_default_branch(default_branch)
```

Comment:
Dang-nabbit black, I was hoping that change would decrease the number of lines used, not make it longer!
I hate to ask for another change here, but I am thinking, given line-length issues, that it might be clearer to change the `then-if-else` _expression_ to just two lines:
```
# Please add a comment here explaining why this had to be done.... 
if os.getenv("GITHUB_ACTIONS") == "true":
    default_branch = f"origin/{default_branch}"
```

The request for comment is because it won't be obvious to someone reading the code why that is there,
and digging back through commits to find the PR and the PR comment is high enough annoyance factor...
Fair to link to the PR comment though, so far as I know, github doesn't purge those for merged commits?